code/modules/atmospherics/machinery/portable/canister.dm

```diff
@@ -352,3 +352,14 @@
 				holding = null
 				. = TRUE
 	update_icon()
+	
+	/obj/machinery/portable_atmospherics/canister/proto
```

Comment:
Robustin, please, just for once, look at the line I've commented on for more than a femtosecond.